src/app/shared/components/notification-box/notification-box-item.component.html

```diff
@@ -27,13 +27,6 @@ <h4 matLine class="mat-list-item-content">
     </span>
   </h4>
   <span>
-    <mat-icon
-      *ngIf="notification.status !== notificationStatus.Pending"
-      (click)="remove()"
-      matListIcon
-      class="close mdi-close"
-    >
-    </mat-icon>
     <mat-icon
       *ngIf="notification.status === notificationStatus.Pending"
       class="close"
```

Comment:
Do we need this code?
See behavior in the notification box. Maybe we will need to add some styles.